entity-management-web/pom.xml

```diff
@@ -261,6 +266,10 @@
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-surefire-plugin</artifactId>
 				<!-- <version>2.22.0</version> -->
+				<configuration>
+        			<reuseForks>false</reuseForks>
```

Comment:
The reason was the zoho test, that run in parallel with other tests during maven build and caused problems because it called the zoho service and probably from the docker container some other tests where running in and therefore could not complete properly. I removed it.